prime-router/docs/universal-pipeline/receive.md

```diff
@@ -1,11 +1,32 @@
 # Receive
+`Receive` is the first step a message goes through when entering the Universal Pipeline (UP). Despite its name, this step
+actually occurs in the `ReportFunction.kt` file. 
 
+There are two calls that will kick off this function:
+```
+POST /api/reports
+POST /api/waters
+```
 
-The UP can receive data in either HL7 or FHIR. We are planning to support CSV in the future.
-## Validation
+`\reports` - old API, does not really have security other than checking that the sender exists. 
+Still widely used.
+
+`\waters` - new API, uses FHIR auth security
 
-Profiles - define implementation guides, give examples
+Currently, the UP can receive data in either HL7 or FHIR. We are planning to support CSV in the future.
 
-## Code Mapping
+## Async/Sync
```

Comment:
The Universal Pipeline is always asnyc. This function is shared with the legacy pipeline which has the option for sync.